stubs/txredisapi.pyi

```diff
@@ -62,6 +62,9 @@ def lazyConnection(
 class ConnectionHandler(RedisProtocol):
     def disconnect(self) -> "Deferred[None]": ...
 
+class UnixConnectionHandler(ConnectionHandler):
+    def __repr__(self) -> str: ...
```

Comment:
Both of these define `__repr__`:

```suggestion
class ConnectionHandler(RedisProtocol):
    def disconnect(self) -> "Deferred[None]": ...
    def __repr__(self) -> str: ...

class UnixConnectionHandler(ConnectionHandler):
    ...
```

Reply:
I'm fine with moving this, but I have an additional question that is related: is there any value in including the other functions that are part of the `ConnectionHandler` class?